manifests/init.pp

```diff
@@ -261,8 +271,19 @@
   }
 
   if $::osfamily == 'RedHat' {
+    if $package_pcs == undef {
+      $_package_pcs = true
```

Comment:
Afaik, variable names starting with _ are no more allowed in Puppet4. This actually refers also to existing variables, not only the ones in this commit.